src/entities/sprint.ts

```diff
@@ -0,0 +1,34 @@
+import { ReducedAgile, ReducedAgileImpl } from "./agile";
+import { ReducedIssue, ReducedIssueImpl, Issue, IssueImpl } from "./issue"
+
+export class ReducedSprintImpl {
+    id?: string = '';
+    name?: string = '';
+    goal?: string = '';
+    start?: number = 0;
+    finish?: number = 0;
+    archived: boolean = false;
+    agile?: ReducedAgile = new ReducedAgileImpl();
```

Comment:
This causes an infinite loop since ReducedAgileImpl tries to instantiate a ReducedSprintImpl. Maybe move this property to the SprintImpl class.